scan_counter.py

```diff
@@ -18,13 +18,15 @@
 
 from config import *
 
-if os.name == "posix":
-    from counter_api_linux import CounterAPI
-elif os.name == "nt":
-    from counter_api_windows import CounterAPI
-else:
-    raise Exception("Operating System not Supported! Must be one of POSIX or NT")
-
+try:
+    if os.name == "posix":
+        from counter_api_linux import CounterAPI
+    elif os.name == "nt":
+        from counter_api_windows import CounterAPI
+    else:
+        raise Exception("Operating System not Supported! Must be one of POSIX or NT")
+except:
+    print("uldaq not installed, must proceed with mock data")
```

Comment:
Nice, I like this ... having to install uldaq to test the software outside of a lab was unreasonably painful.